### PR TITLE
Create Code style guide.md

### PR DESCRIPTION
**Pull request by aveline**
First draft of a code style guide to help with consistency

### CODE REVIEW

File: documentation/Code style guide.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- Use `try-catch` blocks to handle exceptions and errors.
+- Provide meaningful error messages that help developers understand the issue.
+- Log errors appropriately for debugging and troubleshooting purposes.
+
+## Code Organization and Structure
+
+- Organize code files and directories logically and consistently.
+- Use meaningful and descriptive file and directory names.
+- Follow the recommended folder structure for components, utilities, and tests.
+- Import modules and components in alphabetical order.
```
Comment (sam-b-rose) [inline documentation/Code style guide.md:37]: We do already use the [ESLint import/order](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/order.md) rule, which looks like it has an [option for alphabetizing](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/order.md#alphabetize-order-ascdescignore-orderimportkind-ascdescignore-caseinsensitive-truefalse). Can we should enable this and remove this point?

File: documentation/Code style guide.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- Group related functions and variables together within a file.
+- Include JSDoc tags to describe default prop values
```
Comment (sam-b-rose) [inline documentation/Code style guide.md]:
```suggestion
- Include [JSDoc tags](https://jsdoc.app/) to describe default prop values
```

**Review by sam-b-rose — APPROVED** [review]
Excellent start! Thank you for adding this much-needed and long-overdue resource 🙌

File: documentation/Code style guide.md (hunk 1 above)
Comment (aveline) [reply documentation/Code style guide.md:37]: I'm all for it

**Pull request merged**